docs/pages/reference/helm-reference/teleport-cluster.mdx

```diff
@@ -1143,6 +1143,8 @@ See https://kubernetes.io/docs/concepts/configuration/secret/#tls-secrets for mo
 
 `tls.existingCASecretName` sets the `SSL_CERT_FILE` environment variable to load a trusted CA or bundle in PEM format into Teleport pods.
 This can be set to inject a root and/or intermediate CA so that Teleport can build a full trust chain on startup.
+This can also be used to trust private CAs when contacting an OIDC provider, a s3 compatible backend, or any external service without
```

Comment:
```suggestion
This can also be used to trust private CAs when contacting an OIDC provider, an S3-compatible backend, or any external service without
```
Fixes capitalization and noun/article agreement